pyanaconda/exception.py

```diff
@@ -277,7 +277,6 @@ def initExceptionHandling(anaconda):
                   programVersion=startup_utils.get_anaconda_version_string(),
                   programArch=os.uname()[4],
                   attrSkipList=["_intf._actions",
-                                "_intf._currentAction._xklwrapper",
```

Comment:
Ar we sure this is not needed ? This seems to be the list of things to exclude from the python-meh dump, so it might be rather hard to track down if stuff is needed or not, especially as it's all just strings.

Reply:
Kept.